frontend/src/api/types.ts

```diff
@@ -26,12 +26,12 @@ export type ParsedExcelDto = {
   compounds: ParsedCompoundDto[];
 };
 
-export type ResultDto =
+export type ResultDto<T = number[]> =
```

Comment:
:POG: ARE YOU TYPESCRIPT EXPERT?

Reply:
Just looked at `interface Dto<T, Extra = undefined> {` and did something similar :slightly_smiling_face: